src/header/Header.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Link, Route, Routes } from "react-router-dom";
+import Home from "../pages/home/Home";
+import Examples from "../pages/examples/Examples";
+import Blog from "../pages/blog/Blog";
+import About from "../pages/about/About";
+import Login from "../pages/login/Login";
+import Form from "../pages/examples/form/Form";
+import Chat from "../pages/examples/messenger/Chat";
+import "../style/header.css";
+
+const Header = () => {
+  return (
+    <div>
+      <nav className="navbar">
+        <ul className="nav-links">
+          <li className="nav-links__li">
+            <Link className="nav-links__a" to="/">
+              Home Page
+            </Link>
+          </li>
+          <li className="nav-links__li">
+            <Link className="nav-links__a" to="/examples">
+              Examples
+            </Link>
+          </li>
+          <li className="nav-links__li">
+            <Link className="nav-links__a" to="/blog">
+              Blog
+            </Link>
+          </li>
+          <li className="nav-links__li">
+            <Link className="nav-links__a" to="/about">
+              About
+            </Link>
+          </li>
+          <li className="nav-links__li">
+            <Link className="nav-links__a" to="/login">
+              Login
+            </Link>
+          </li>
+        </ul>
+      </nav>
+
+      <Routes>
```

Comment:
https://reactrouter.com/en/main/router-components/browser-router
You need to move the routing to a separate folder and make it more isolated. Also, why not write a loop that would go through the array and return the same thing as here?
Advice: If it is possible to automate a process, automate it